build/ci/ego

```diff
@@ -17,8 +17,10 @@
 set -o nounset
 set -o errexit
 
-readonly DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-readonly SCRIPT=$(basename "${BASH_SOURCE[0]}")
+readonly DIR=
```

Comment:
I think the llinter for this got updated and it now complains, happy to make a separate PR